config/experimental/s3configsource/config.go

```diff
@@ -0,0 +1,19 @@
+package s3configsource
+
+import (
+	splunkprovider "github.com/signalfx/splunk-otel-collector"
```

Comment:
You want to replace with config/experimental/configprovider, right?

Reply:
Yes, which will eventually be fully merged in